$NetBSD: patch-configure,v 1.1 2021/05/25 10:44:22 thor Exp $

Workaround for pkgsrc's libtool to not build static libs when not 
requested. This patch should vanish on updating to > 1.27.2.

--- configure.orig	2021-05-23 00:04:28.468168350 +0000
+++ configure
@@ -5902,9 +5902,9 @@ else
   enable_static=no
 fi
 
-
-
-
+if test "$enable_static" = no; then
+  LT_LDFLAGS="$LT_LDFLAGS -shared"
+fi
 
 
 
